Jenkinsfile

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent none
+  environment {
+    BASE_DIR="src/github.com/elastic/hey-apm"
+    APM_SERVER_BASE_DIR = "src/github.com/elastic/apm-server"
+    JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
+    GO_VERSION = "${params.GO_VERSION}"
+    APM_SERVER_VERSION = "${params.APM_SERVER_VERSION}"
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS') 
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+  }
+  parameters {
```

Comment:
out of curiosity, what is the difference between these `parameters` and https://github.com/elastic/hey-apm/pull/69/files#diff-7a8b8e4479c1b69215ddb619ce195f8fR23?

Reply:
The Jenkinsfile is used by a multibranch pipeline job so do not need to specify a branch to build, for each branch/PR/tag will exist a project to run it.